README.md

```diff
@@ -1,173 +1,132 @@
+## Version Support
+| SDK Version   | Vue 2    | Vue 3  |
+|---------------|----------|--------|
+| 1.0.0 - 1.2.3 | V        | X      |
 
-# 🛠️ Installation 
-## Vue 2.x
-1. Install using Vue-CLI
-    - After you create your application with Vue-CLI, navigate to the created app folder, and install Cloudinary SDK by:
-        ```bash
-        vue add cloudinary
-        ```
-    - Set up your cloudName and pick the components to use (or use all 😃)
-    ![Set up with cloudName and options](http://bit.ly/2WSKTf0)
-    - A `cloudinary.js` file will be added to your src directory, same level with your `main.js` file. Feel free to customize it if you like. And have fun with Cloudinary! 🤟
-    - More information on the plugin, check out [our Vue-CLI plugin for Cloudinary Repo](https://github.com/cloudinary/vue-cli-plugin-cloudinary)
 
-2. install using your favorite package manager (yarn, npm)
+
+## Installation
+### Install using Vue-CLI
+- After you create your application with Vue-CLI, navigate to the created app folder, and install Cloudinary SDK by:
     ```bash
-    npm install cloudinary-vue
-    yarn add cloudinary-vue
+    vue add cloudinary
     ```
-
+- Set up your cloud name and pick the components to use (or use all 😃)
+  ![Set up with cloudName and options](http://bit.ly/2WSKTf0)
+- A `cloudinary.js` file will be added to your src directory, same level with your `main.js` file. Feel free to customize it if you like. And have fun with Cloudinary! 🤟
+- More information on the plugin, check out [our Vue-CLI plugin for Cloudinary Repo](https://github.com/cloudinary/vue-cli-plugin-cloudinary)
+
+### install using your favorite package manager (yarn, npm)
+```bash
+npm install cloudinary-vue
+yarn add cloudinary-vue
+```
 ### Nuxt.js
-
 Please use [Cloudinary module](https://cloudinary.nuxtjs.org/) for [Nuxt.js](https://nuxtjs.org) projects. You can set it up by using the following:
-
 ```
 yarn add @nuxtjs/cloudinary
 #OR
 npm i @nuxtjs/cloudinary
 ```
 
-## Setup and configuration
+## Usage 
+### Setup
+```javascript
+import Vue from 'vue';
+import Cloudinary, { CldImage, CldTransformation } from "cloudinary-vue";
+
+Vue.use(Cloudinary, {
+  configuration: { cloudName: "demo" },
+  components: {
+    CldImage,
+    CldTransformation
+  }
+});
+```
+You can also import the cloudinary components manually in each of your components.
 
-### Vue 2.x - Setup and configuration
-1. **A Global setup** - Include CloudinaryVue globally
-    ```javascript
-    import Vue from 'vue';
-    import Cloudinary, { CldImage, CldTransformation } from "cloudinary-vue";
+### Transform and Optimize Assets 
+- [See full documentation](https://cloudinary.com/documentation/vue_integration#installation_and_setup)
     
-    Vue.use(Cloudinary, {
-      configuration: { cloudName: "demo" },
-      components: {
-        CldImage,
-        CldTransformation
-      }
-    });
+   ```jsx
+    // Apply a single transformation
+        <cld-context cloudName="demo">
```

Comment:
Is the cld-context missing from the setup section or will this work?

Reply:
Right, updated the setup!